azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -93,7 +93,7 @@ private void sendSlaAlertEmail(SlaOption slaOption, String slaMessage) {
           super.createEmailMessage(subject, "text/html", emailList);
 
       message.setBody(body);
-
+      message.setTLS(tls);
```

Comment:
Right, you did a better job if considered other bugs like this.

But, it seems no necessary to call the statement

> message.setTLS(tls)

at the public method **sendFirstErrorMessage** and **sendErrorEmail** in azkaban-common/src/main/java/azkaban/utils/Emailer.java based on your fixes.

I don't know why the previously submit did this things. maybe you can remove the statement from the method sendFirstErrorMessage and the other likely method in azkaban-common/src/main/java/azkaban/utils/Emailer.java in your fixes? 
Or just use the fixes in this issue to keep consistency with previously submit code ?